Automattic/sensei
# Separate notices and errors in the Done page.

Part of #3337

The second and final part is in https://github.com/Automattic/sensei/pull/3394

### Changes proposed in this Pull Request

* This PR restructure the Done page to the new format, separating notices and errors.
* It still doesn't have titles and links. It'll be done in another PR, so this PR doesn't get bigger.

### Testing instructions

* Go to the importer.
* Import Donna's files.
* Check that now there are different areas for the notices and the errors in the done step.
* Simulate an error - I tested renaming the route from `$this->rest_base . '/(?P<job_id>[0-9a-z]+)/logs',` to `$this->rest_base . '/(?P<job_id>[0-9a-z]+)/logsX',` in the `includes/rest-api/class-sensei-rest-api-data-port-controller.php` file).
* Re-import the files and make sure you see an error with the retry button.
* Revert the change, click retry, and make sure the logs are loaded.


### Screenshot / Video

<img width="805" alt="Screen Shot 2020-07-09 at 15 35 45" src="https://user-images.githubusercontent.com/876340/87078081-807ecd80-c1fa-11ea-87b4-aeda24f7846d.png">
